packages/vx-responsive/src/components/ParentSize.js

```diff
@@ -43,9 +43,9 @@ export default class ParentSize extends React.Component {
     this.target = ref;
   }
   render() {
-    const { className, children } = this.props;
+    const { className, children, ...rest } = this.props;

```

Comment:
Super minor nit, but to match the rest of the codebase could we rename this `...restProps`?

Reply:
Done, sorry, we were in a rush at work and we missed this.
Thank you